src/repr/type_binary.ml

```diff
@@ -165,7 +163,7 @@ module Encode = struct
   and map : type a b. boxed:bool -> (a, b) map -> b encode_bin =
    fun ~boxed { x; g; _ } ->
     let encode_bin = unstage (if boxed then t x else unboxed x) in
-    stage (fun u k -> encode_bin (g u) k)
+    stage (fun u oc -> encode_bin (g u) oc)
```

Comment:
`encode_bin` now expects 3 arguments - for consistency can you call them `byt` and `off` as below?